frontend/src/pages/TermPlanner/TermPlanner.tsx

```diff
@@ -167,12 +166,14 @@ const TermPlanner = () => {
 
     if (destination.droppableId === 'unplanned') {
       // === move course to unplanned ===
-      dispatch(
-        unschedule({
-          destIndex,
-          code: draggableId
-        })
-      );
+      try {
+        await axios.post('/planner/unscheduleCourse', JSON.stringify({ courseCode: draggableId }), {
+          params: { token }
+        });
```

Comment:
@leonardo-fan can you verify this is the correct way to make this request since i've also seen it passed straight in without the `params: ` part